synctl

```diff
@@ -132,19 +132,38 @@ def start_worker(app: str, configfile: str, worker_configfile: str) -> bool:
         return False
 
 
-def stop(pidfile, app):
+def stop(pidfile, app) -> bool:
+    """Attempts to kill a synapse worker from the pidfile.
+    Args:
+        pidfile: path to file containing worker's pid
+        app: name of the worker's appservice
+
+    Returns:
+        True if the process stopped successfully
+        False if process was stopped or an error occured
```

Comment:
This is a bit confusing -- does this return true or false if the process stops?

Reply:
I simply added "if process was **already** stopped". It makes it a bit clearer.